buildspec_load_test.yml

```diff
@@ -0,0 +1,50 @@
+version: 0.2
+env:
+  shell: bash
+  variables: 
+    THROUGHPUT_LIST: '["10m", "20m", "30m"]'
+    CW_THROUGHPUT_LIST: '["1m", "2m", "3m"]'
+    TESTING_RESOURCES_STACK_NAME: 'load-test-fluent-bit-testing-resources'
+    LOG_STORAGE_STACK_NAME: 'load-test-fluent-bit-log-storage'
+    PREFIX: 'load-test-fluent-bit-'
+
+phases:
+  install:
+    runtime-versions:
+      golang: 1.13
+      python: 3.x
+      nodejs: 12
+  pre_build:
+    commands:
+      - echo Running the load tests
+      - npm config set prefix /usr/local
+      - npm install -g aws-cdk@latest
+      - curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+      - unzip awscliv2.zip
+      - which aws
+      - ls -l /root/.pyenv/shims/aws
+      - ./aws/install --bin-dir /root/.pyenv/shims --install-dir /root/.pyenv/shims --update
+      - aws configure set default.region us-west-2
+  build:
+    commands:
+      # Assume role to get the credentials, and set them as environment variables
+      - CREDS=`aws sts assume-role --role-arn ${CFN_DEPLOY_ROLE_ARN} --role-session-name test-cfn`
```

Comment:
We may want a more understandable name here for `role-session-name`. This naming is just for my testing and we can update it after we generate one in our account.

Reply:
may change to `aws sts assume-role --role-arn ${LOAD_TEST_CFN_ROLE_ARN} --role-session-name loadTestCfn` for now

Reply:
Will change to --role-arn ${LOAD_TEST_CFN_ROLE_ARN} --role-session-name load-test-cfn